ydb/docs/en/core/reference/kafka-api/examples.md

```diff
@@ -93,36 +93,237 @@ To configure [Fluent Bit](https://github.com/fluent/fluent-bit), use the followi
+
+{% note warning %}
+
+Kafka Connect instances for working with YDB should only be deployed in standalone mode. YDB does not support Kafka Connect in distributed mode.
+
+{% endnote %}
+
+The actual data movement is performed using connectors that run in separate threads of the executing process.
+
+For more information about Kafka Connect and its configuration, see the [Apache Kafka®](https://kafka.apache.org/documentation/#connect) documentation.
+
+### Setting up Kafka Connect
+
+1. [Create a consumer](../ydb-cli/topic-consumer-add.md) with the name `connect-<connector-name>`. The connector name is specified in the configuration file when you set it up in the `name` field.
+
+1. [Download](https://downloads.apache.org/kafka/) and unzip the Apache Kafka® archive:
+
+    ```bash
+    wget https://downloads.apache.org/kafka/3.6.1/kafka_2.13-3.6.1.tgz && tar -xvf kafka_2.13-3.6.1.tgz --strip 1 --directory /opt/kafka/
+    ```
+
+    This example uses Apache Kafka® version ``3.6.1``.
+
+1. Create a directory with the executable process settings:
+
+    ```bash
+    sudo mkdir --parents /etc/kafka-connect-worker
+    ```
+
+1. Create the executor process settings file `/etc/kafka-connect-worker/worker.properties'
+
+    ```ini
+    # Main properties
+    bootstrap.servers=ydb:9093
+
+    # AdminAPI properties
+    sasl.mechanism=PLAIN
+    security.protocol=SASL_SSL
+    sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="<user>@<db>" password="<user-pass>";
+
+    # Producer properties
+    producer.sasl.mechanism=PLAIN
+    producer.security.protocol=SASL_SSL
+    producer.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="<user>@<db>" password="<user-pass>";
+
+    # Consumer properties
+    consumer.sasl.mechanism=PLAIN
+    consumer.security.protocol=SASL_SSL
+    consumer.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="<user>@<db>" password="<user-pass>";
+
+    consumer.partition.assignment.strategy=org.apache.kafka.kafka.clients.consumer.RoundRobinAssignor
+    consumer.check.crcs=false
+
+    # Converter properties
+    key.converter=org.apache.kafka.connect.storage.StringConverter
+    value.converter=org.apache.kafka.connect.storage.StringConverter
+    key.converter.schemas.enable=false
+    value.converter.schemas.enable=false
+
+    # Worker properties
+    plugin.path=/etc/kafka-connect-worker/plugins
+    offset.storage.file.filename=/etc/kafka-connect-worker/worker.offset
+    ```
+
+1. Create a FileSink connector settings file `/etc/kafka-connect-worker/file-sink.properties` to move data from the YDB topics to the file:
+
+    ```ini
+    name=local-file-sink
+    connector.class=FileStreamSink
+    tasks.max=1
+    file=file_to_write.json
+    topics=Topic-1
+    ```
+
+    Where:
+
+    * `file` - name of the file to which the connector will write data.

```

Comment:
it's not clear where this file will reside, and also can it be a relative/absolute path?

Reply:
Changed it to an absolute path.